projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -327,15 +327,23 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
             // manually trigger text selection as it will not be triggered during editing
             this.textSelection.trigger();
             return;
-        }
-        if (event.key === this.platformUtil.KEYMAP.BACKSPACE
+        } else if (event.key === this.platformUtil.KEYMAP.BACKSPACE
             || event.key === this.platformUtil.KEYMAP.DELETE) {
             this._updateInput = false;
             this.clearSelection(true);
-        }
-        if (!this.collapsed && event.key === this.platformUtil.KEYMAP.TAB) {
+        } else if (event.key === this.platformUtil.KEYMAP.TAB) {
             this.clearOnBlur();
             this.close();
+        } else {
+            if (event.key !== this.platformUtil.KEYMAP.ARROW_DOWN && event.key !== this.platformUtil.KEYMAP.SPACE) {
```

Comment:
should not filter out `SPACE` here. I can filter values like `New Mexico`, `New York`, `New Jersey` and so on by typing `SPACE` + `M` and combo should show first all the items containing space, then items containing space and m charecter.